pages/api-reference/connect/transfers/initiate.mdx

```diff
@@ -0,0 +1,161 @@
+---
+layout: guides
+product: connect
+title: Initiate a Transfer
+guide:
+  step: 1
+meta:
+  title: Initiate a Transfer | Dwolla Connect | Dwolla API Reference
+  description: Endpoint for initiating a transfer.
+---
+
+import InitiateTransfer from "../../../../assets/api-resource-tables/transfers/InitiateTransfer";
+
+# Initiate a transfer
+
+This section details how to initiate a transfer between your main Dwolla Account and external-parties.
+
+### Prevent duplicate transfers with idempotency key
+
+To prevent an operation from being performed more than once, Dwolla supports passing in an `Idempotency-Key` header with a unique key as the value. Multiple `POSTs` with the same idempotency key won't result in multiple resources being created.
+
+For example, if a request to initiate a transfer fails due to a network connection issue, you can reattempt the request with the same idempotency key to ensure that only a single transfer is created.
+
+Refer to our [idempotency key](https://developers.dwolla.com/api-reference#idempotency-key) section to learn more.
+
+## HTTP request
+
+> `POST https://api.dwolla.com/transfers`
+
+## Request parameters
+
+<Table table={InitiateTransfer} />
+
+## HTTP status and error codes
+
+| HTTP Status | Error Message             | Description                                                                                    |
+| ----------- | ------------------------- | ---------------------------------------------------------------------------------------------- |
+| 201         | Created.                  | A transfer was created.                                                                        |
+| 400         | Funding source not found. | Double check the funding source Id, and make sure you are using the correct funding source Id. |
+| 400         | Invalid amount            | The supplied amount must be a positive number.                                                 |
+| 401         | Invalid access token      | Access token not valid. Generate a new one and try again.                                      |
+| 403         | Forbidden                 | Not authorized to create a transfer.                                                           |
+
+## Request and response - using `ach`
```

Comment:
Let's remove `- using ach` for now until we have more transfer types to demonstrate in the docs